hollow/src/main/java/com/netflix/hollow/core/write/HollowWriteFieldUtils.java

```diff
@@ -0,0 +1,26 @@
+package com.netflix.hollow.core.write;
+
+import com.netflix.hollow.core.memory.ByteData;
+
+public class HollowWriteFieldUtils {
+
+    public static int readIntBits(ByteData data, long fieldPosition) {
```

Comment:
These two methods could be placed as default methods on ByteData, which then allows for more optimal implementations by overriding implementations (if present)